src/containers/tutor-profile/about-tutor-block/AboutTutorBlock.tsx

```diff
@@ -21,6 +24,13 @@ const AboutTutorBlock = () => {
     setActiveIndex(activeIndex !== index ? index : null)
   }
 
+  const accordionItems = Object.keys(data)
+    .filter((el) => data[el as keyof ProfessionalBlock])

```

Comment:
Is `as keyof ProfessionalBlock` the best solution here? I think you can move `Object.keys(data)` to separate variable and type it instead, because `Object.keys` are not super type safe

Reply:
I didn't like it there too, thanks for the idea! Fixed